assets/js/frontend/user-registration.js

```diff
@@ -2,16 +2,72 @@
 /* global  ur_google_recaptcha_code */
 /* global  grecaptcha */
 (function ($) {
+	var passwords_not_matched = false;
+
 	var user_registration = {
 		$user_registration: $('.ur-frontend-form form.register'),
 		init: function () {
 			this.load_validation();
 			this.init_inputMask();
 			this.init_tiptip();
+			this.init_real_time_password_match();
 
 			// Inline validation
 			this.$user_registration.on('input validate change', '.input-text, select, input:checkbox input:radio', this.validate_field);
 		},
+		init_real_time_password_match: function () {
+			let error_message = `<p class="user-registration-error" id="password-not-matched-error">Passwords doesn't match!</p>`;
```

Comment:
Please Localize error message and make error message HTML structure consistent like others use <label> tag instead of <p> tag.

Reply:
Okay. I'll work on it right away.